com_redhat_kdump/gui/spokes/kdump.py

```diff
@@ -50,12 +51,12 @@ class KdumpSpoke(NormalSpoke):
 
     @classmethod
     def should_run(cls, environment, data):
-        # the KdumpSpoke should run only if requested
-        return kernel_arguments.is_enabled("kdump_addon")
+        return is_module_available(KDUMP)
```

Comment:
Hi, previously this addon only work if kdump_addon is specified in kernel cmdline, will this change the behavior?

Reply:
Ah, never mind, I saw the kernel_arguments.is_enabled("kdump_addon"): in the service py file below.